src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/PreviewToken.cs

```diff
@@ -22,7 +22,7 @@ public class PreviewToken : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOf
 
     public required DateTimeOffset Activates { get; set; }
 
-    public required DateTimeOffset Expiry { get; set; }
+    public required DateTimeOffset Expires { get; set; }
```

Comment:
I think there needs to be a Public API database migration added for this rename.